src/morphology/english/getStem.js

```diff
@@ -0,0 +1,157 @@
+import { buildOneFormFromRegex } from "../morphoHelpers/buildFormRule";
```

Comment:
This whole file was renamed to `determineStem.js`. Please figure out how this came to live here and fix it 😅

Reply:
I guess it's a result of renaming + unfortunate version control. This file is never called, so I just unfancily deleted it.